ferrocene/tools/traceability-matrix/src/test_outcomes.rs

```diff
@@ -229,109 +266,71 @@ mod tests {
                 },
             ],
         })
-        .to_string();
-        let content: serde_json::Value = serde_json::from_str(&content).unwrap();
-        // for ease of debugging
-        let content = serde_json::to_string_pretty(&content).unwrap();
-        std::fs::write(dir.path().join("runner1.json"), content).unwrap();
```

Comment:
@pietroalbini Before, the test 1) created a `serde_json::Value` with `json!`, then 2) transformed that into a `String` with `.to_string`, then 3) transformed it into a `serde_json::Value` with `serde_json::from_str` and then 4) transformed it into a (formatted) `String` with `serde_json::to_string_pretty`.

Is there a reasong it has to go `Value -> String -> Value -> String`, or can we just go from `Value` to a formatted `String`, directly? (the direct way is what I did for this PR)

Reply:
Also, is the pretty formatting still necessary? It states it is "for ease of debugging", but I guess in case we are debugging this test, we can also format the file manually, can't we?

Reply:
> what is the reason to remove it

It seems unnecessary and would simplify the code.